packages/kbn-test/src/auth/sesson_manager.test.ts

```diff
@@ -117,6 +134,15 @@ describe('SamlSessionManager', () => {
 
     test(`doesn't throw error when supportedRoles is not defined`, async () => {
       const nonExistingRole = 'tester';
+      const testData: UserProfile = {
+        username: '6ta90xc',
```

Comment:
Same as above?  What is this value?

Reply:
just random string, user has no control over it.

Reply:
ah got it, thanks mate!